icu4c/source/i18n/messageformat2_function_registry_internal.h

```diff
@@ -39,7 +39,7 @@ namespace message2 {
 
         private:
         static UnicodeString getStringOption(const FunctionOptions& opts,
-                                             const UnicodeString& optionName,
+                                             const std::u16string_view optionName,
```

Comment:
Please remove the `const` from the string_view-by-value parameter, here and in the other functions.

Reply:
Fixed in [8500b13](https://github.com/unicode-org/icu/pull/3418/commits/8500b1344d108b4d471d973448b06146fc741acb)